package.json

```diff
@@ -1,170 +1,39 @@
-    "@types/express-session": "^1.17.0",
-    "@types/jest": "^24.9.1",
-    "@types/lodash": "^4.14.165",
-    "@types/morgan": "^1.9.1",
-    "@types/node": "^12.12.54",
-    "@types/passport": "^1.0.4",
-    "@types/react": "^16.9.48",
-    "@types/react-dom": "^16.9.8",
-    "@types/react-plotly.js": "^2.2.4",
-    "@types/react-redux": "^7.1.9",
-    "@types/react-router-dom": "^5.1.5",
-    "@types/tough-cookie": "^4.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.1.1",
-    "@typescript-eslint/parser": "^4.1.1",
-    "concurrently": "^5.3.0",
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-airbnb-typescript": "^10.0.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.2",
-    "node-sass": "^4.14.1",
-    "nodemon": "^2.0.4",
-    "openssl": "^1.1.0",
-    "ts-node": "^9.0.0",
-    "typescript": "^3.7.5"
+    "node": ">=14",
+    "npm": ">=7",
+    "yarn": ">=1"
   },
+  "workspaces": [
+    "client",
+    "server"
+  ],
   "scripts": {
     "start": "./run-prod.sh",
     "dev": "./run-dev.sh",
-    "build": "./build-prod.sh",
     "test": "./run-tests.sh",
+    "build": "./build.sh",
     "start-db": "./start-db.sh",
     "stop-db": "./stop-db.sh",
-    "seed-dev": "./seed-dev.sh",
-    "build-run-dev": "./build-run-dev.sh",
-    "typeorm": "ts-node --project ./server/tsconfig.json ./node_modules/.bin/typeorm",
-    "migration-create": "./migration-create.sh",
-    "migration-generate": "./migration-generate.sh",
-    "migration-run": "./migration-run.sh",
-    "migration-revert": "./migration-revert.sh"
+    "seed-dev": "cd server && ./seed-dev.sh",
+    "migration-create": "cd server && ./migration-create.sh",
+    "migration-generate": "cd server && ./migration-generate.sh",
+    "migration-revert": "cd server && ./migration-revert.sh",
+    "migration-run": "cd server && ./migration-run.sh"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "proxy": "http://127.0.0.1:4000/",
-  "rescripts": [
-    [
-      "use-babel-config",
-      {
-        "presets": [
-          "react-app",
-          [
-            "@babel/preset-typescript",
-            {
-              "allowNamespaces": true
-            }
-          ]
-        ]
-      }
-    ]
-  ]
+  "dependencies": {
+    "concurrently": "^6.2.0",
+    "eslint": "^7.29.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb-typescript": "^12.3.1",
+    "eslint-plugin-chai-friendly": "^0.7.1",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-mocha-no-only": "^1.1.1",
+    "eslint-plugin-react": "^7.24.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
The reason for all of the react eslint plugins at this top level is just that `eslint-config-airbnb-typescript` seems to have some hard dependencies on them (oddly). So adding them all here suppressed yarn install warnings.